.github/workflows/release_hotfix.yml

```diff
@@ -26,5 +26,16 @@ jobs:
       run: |
         git config user.name "ami-ci"
         git config user.email "<>"
+    
+        # Update the package.json version. Skip the git tag, we'll do that later
+        npm version ${{ steps.bump-semver.outputs.new_version }} --no-git-tag-version
+    
+        # Commit the changed package.json file to master
+        git add package.json
+        git commit -m "Bump version to ${{ steps.bump-semver.outputs.new_version }}"
+        git push origin master
```

Comment:
To avoid unnecessary builds, see: https://github.com/customerio/gist-web/blob/develop/.github/workflows/build.yml shall we add [skip ci] in the commit message?